src/js/mockObject.js

```diff
@@ -0,0 +1,23 @@
+export default function mockObject() {
+  const array = [];
+
+  for (let index = 0; index < 10; index++) {
+    array.push({
+      id: {
+        videoId: index,
+      },
+      snippet: {
+        channelTitle: "essential;",
+        thumbnails: {
+          high: {
+            url: "https://i.ytimg.com/vi/ECfuKi5-Cfs/hq720.jpg?sqp=-oaymwEcCOgCEMoBSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLDvmIcX-TgdcH2g_Bd4AUxw6hjmvQ",
+          },
+        },
+        publishTime: "2022-03-02T11:39:31Z",
+        title: "[Playlist] 너무 좋은데 괜찮으시겠어요?",
+      },
+    });
+  }
+
+  return array;
```

Comment:
```suggestion
  return [...Array(10).keys()].map((index) => ({
        id: {
          videoId: index,
        },
        snippet: {
          channelTitle: "essential;",
          thumbnails: {
            high: {
              url: "https://i.ytimg.com/vi/ECfuKi5-Cfs/hq720.jpg?sqp=-oaymwEcCOgCEMoBSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLDvmIcX-TgdcH2g_Bd4AUxw6hjmvQ",
            },
          },
          publishTime: "2022-03-02T11:39:31Z",
          title: "[Playlist] 너무 좋은데 괜찮으시겠어요?",
        },
    })
  )
```
아주 조금.. 더 깔끔하게 할 수 있어요

Reply:
😮😮😮... 이렇게 응용할수도 있었네요...!!!